sites/postcss-preset-env/src/static/js/playground.js

```diff
@@ -0,0 +1,197 @@
+import { EditorState, Transaction, Annotation } from '@codemirror/state';
+import { EditorView } from '@codemirror/view';
+import { basicSetup } from 'codemirror';
+import { css } from '@codemirror/lang-css';
+import { javascript } from '@codemirror/lang-javascript';
+import { oneDark } from '@codemirror/theme-one-dark';
+import postcss from 'postcss';
+import postcssPresetEnv from 'postcss-preset-env';
+
+// hack to make browserslist work in a browser. (irony...)
+self.process = { env: {} };
+
+const currentConfig = {
+	browsers: ['> 0.2% and not dead'],
+	minimumVendorImplementations: 0,
+	stage: 2,
+	preserve: null,
+};
```

Comment:
Should we add the logical options too? Could do that myself after too

Reply:
We should!